src/wp-includes/media.php

```diff
@@ -176,9 +176,10 @@ function image_hwstring( $width, $height ) {
  *
  * @since 2.5.0
```

Comment:
Adding a new parameter should always be documented here, like
```suggestion
 * @since 2.5.0
 * @since 6.1.0 Added the $mime_type parameter.
```

Please also add such an additional `@since` annotation for every other function in this PR where you're adding the parameter.